docs/eks-installation.md

```diff
@@ -20,20 +20,14 @@ deployment yaml at:
 https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea-eks.yml
 ```
 
-Based on Kubernetes service cluster IP range, adjust ``serviceCIDR`` values of antrea-agent.conf
-in antrea-eks.yml accordingly, and apply antrea-eks.yml to the EKS cluster.
-
-```bash
-kubectl apply -f antrea-eks.yaml 
 ```
-Now Antrea should be plugged into the EKS CNI and is ready to enforce NetworkPolicy.
+Note: With Antrea >=v0.9 release, you should apply `antrea-eks-node-init.yaml` before deploying Antrea.
+This would restart existing PODs (except in host network) so that Antrea can also manage it, once installed.
 
-### Caveats
-
-Some Pods may already be installed before Antrea deployment. Antrea cannot enforce NetworkPolicy
-on these pre-installed Pods. This may be remedied by restarting the Pods. For example,
+kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea-eks-node-init.yml
+```
 
 ```bash
-kubectl scale deployment coredns --replicas 0 -n kube-system
-kubectl scale deployment coredns --replicas 2 -n kube-system
+kubectl apply -f antrea-eks.yaml
 ```
+Now Antrea should be plugged into the EKS CNI and is ready to enforce NetworkPolicy.
```

Comment:
what is plugged? do you mean "Now Antrea should be chained with the EKS CNI..."

Reply:
I didnt add this line. It was there in prior documentation. And I think it meant Antrea is plugged with EKS CNI